service/storage_proxy.cc

```diff
@@ -2791,11 +2791,11 @@ void storage_proxy_stats::stats::register_stats() {
 
         sm::make_total_operations("received_hint_count", received_hint_count,
                         sm::description("number of hints received by this node"),
-                       {storage_proxy_stats::current_scheduling_group_label()}).set_skip_when_empty(false),
+                        {storage_proxy_stats::current_scheduling_group_label()}).set_skip_when_empty(),
 
         sm::make_total_bytes("received_hint_total_size", received_hint_total_size,
                         sm::description("total size of hints received by this node"),
-                       {storage_proxy_stats::current_scheduling_group_label()}).set_skip_when_empty(false),
+                        {storage_proxy_stats::current_scheduling_group_label()}).set_skip_when_empty(),
```

Comment:
Ah, you removed `.set_skip_when_empty(false)`, but you did it in the wrong commit.